patterns/definitions.owl

```diff
@@ -12640,6 +12640,7 @@ Declaration(Class(<http://purl.obolibrary.org/obo/OBA_2050415>))
 Declaration(Class(<http://purl.obolibrary.org/obo/OBA_2050416>))
 Declaration(Class(<http://purl.obolibrary.org/obo/OBA_2050417>))
 Declaration(Class(<http://purl.obolibrary.org/obo/OBA_2050418>))
+Declaration(Class(<http://purl.obolibrary.org/obo/OBA_2050419>))
```

Comment:
@matentzn , Is this addition necessary in this PR?

Reply:
Not necessary, it just happened when I ran the test. Feel free to remove.